app/src/main/java/com/a9ts/a9ts/main/MainFragment.kt

```diff
@@ -0,0 +1,29 @@
+package com.a9ts.a9ts.main
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.a9ts.a9ts.model.AppointmentRepository
+import com.a9ts.a9ts.databinding.MainFragmentBinding
+
+class MainFragment : Fragment() {
+    lateinit var binding: MainFragmentBinding
+
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+
+        binding = MainFragmentBinding.inflate(layoutInflater)
+
+
+        val adapter = AppointmentListAdapter(AppointmentRepository.appointmentList)
```

Comment:
It is easy when it is a static list. When we will download it from the internet, we will need some repo functions, either coroutines or callbacks, and to make it out of the book, view models :)